api/OEP_API_tutorial_part1.ipynb

```diff
@@ -609,9 +617,16 @@
    "metadata": {},
    "source": [
     "<div class=\"alert alert-block alert-info\">\n",
-    "**200** succesfully changed data!\n",
+    "<b>200</b> succesfully changed data!\n",
     "</div>"
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
   }
```

Comment:
Should we always have an empty cell at the end of the notebooks? I would personally vote for no